user-guide/Basic_Functionality/Visio/turning_shapes_into_controls/Making_a_shape_filter_Alarm_Console_tabs_when_clicked.md

```diff
@@ -39,6 +41,8 @@ Add a shape data field of type AlarmFilter to the shape, and enter an alarm filt
   "Service Impact"<2
   ```
 
+- Example dynamic value
```

Comment:
@KlaasSL, could you provide me with an example of an alarm filter using a dynamic value that I can add to this overview of examples?

```txt
- To filter out all alarms .... :
   \```txt
   [add dynamic value here]
   \```
```

Reply:
Example "[var:myAlarmFilterVariable]"